src/components/ModelSteps/Information.jsx

```diff
@@ -118,15 +118,11 @@ const Information = ({ currentStep }) => {
           </div>
         ))}
       />
-      <DialogActions
-        disableSpacing
-        className={'columnLayout u-mh-0 u-mb-1 cozyDialogActions'}
-      >
+      <DialogActions disableSpacing className={'u-mh-0 u-mb-1'}>
         <Button
-          className="u-db"
-          extension="full"
           label={t('common.next')}
           onClick={submit}
+          fullWidth
```

Comment:
nit: why not testing attribute, when adding feature in CompositeHeeader, Contact and Information?

Reply:
C'est juste une mise à jour de prop, car en passant par l'import `*/buttons` (au lieu de `*/button`) nous utilisons directement l'API de mui (qui test déjà ses composants)